### ISSUE TITLE
Questions about the recognition and cls dataset needed

### ISSUE THREAD

**Issue by inesriahi** [PaddlePaddle/PaddleOCR#9199]
Hi!
Thanks for this awesome library. I have a couple of questions regarding the dataset I'm trying to fine-tune the recognition model with.

1. I'm extracting text from documents. The segmented detected boxes for many segments might look very similar but not to the extent of pixel to pixel correspondance. There can be many segments that look similar in the dataset. This is a sample:
![image](https://user-images.githubusercontent.com/38370333/220829987-a5e89c53-c236-442f-b3f8-55e4259200ae.png)
Would this harm the model I'm finetuning? or should I write something to detect duplicates and remove them? and what criteria should I consider to decide that 2 images are duplicates?

2. In [finetune_en.md](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.6/doc/doc_en/finetune_en.md), it is written: "If there is general real scene data added, it is recommended that in each epoch, the amount of vertical scene data and real scene data should be kept at about 1:1. For example: your own vertical scene recognition data volume is 1W, the data label file is vertical.txt, the collected general scene recognition data volume is 10W, and the data label file is general.txt."
   Does this apply to english?

3. I have numbers that are detected, but are rotated 90 or 270 degrees. I think this is mostly because width to height ratio in the code. How would I modify this to be corrected and rotated whether 0 or 180 to be entered correctly to the pipeline without harming the classification of the other segments?
Example:
![image](https://user-images.githubusercontent.com/38370333/220830700-284d371f-cfeb-4e4b-843c-8822fc30c568.png)

Thank you in advance.

**Comment by WenmuZhou** [comment]
1. This kind of duplication can be regarded as data enhancement, which can enhance network capabilities
2. yes，but if you don't generate data, this can be omitted
3. You can rotate each picture by 0,90,180,270 degrees and send it to the recognition model

**Comment by inesriahi** [comment]
> 1. This kind of duplication can be regarded as data enhancement, which can enhance network capabilities
> 2. yes，but if you don't generate data, this can be omitted
> 3. You can rotate each picture by 0,90,180,270 degrees and send it to the recognition model

Thank you for your fast reply. 
In (3) Should I finetune it for the whole dataset even though this only occurs for numbers and single letter?
